### PR TITLE
feat: update storage url scheme

### PR DESCRIPTION
**Pull request by haizhou**
Update indico url scheme for blob

### CODE REVIEW

**Comment by goatrocks** [comment]
does this go with #195 ?

**Comment by haizhou** [comment]
> does this go with #195 ?

Yes. I think #195 is an additive feature, but this PR is a breaking change. I was also gonna ask how we manage client sdk version updates.

**Comment by sihrc** [comment]
> > > does this go with #195 ?
> > 
> > 
> > Yes. I think #195 is an additive feature, but this PR is a breaking change. I was also gonna ask how we manage client sdk version updates.
> 
> We can create pre-release versions from a branch using github releases -- the version of the client SDK matches the minimum version of the platform. It sounds like this would go in a 6.0SDK -- and perhaps it also needs to get changed in the C# and Java SDKs as well?

tl;dr, we're not allowing breaking changes, so we're making the API compatible. We can move these changes into the other changes for blob storage.

**Comment by haizhou** [comment]
Updated the merge base to #195
